recipes/sphinx-testing/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "sphinx-testing" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ef661775b5722d7b00f67fc229104317d35637a4fb4434bf2c005afdf1da4d09"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - six
+    - sphinx
+  run:
+    - python
+    - six
+    - sphinx
+
+test:
+  imports:
+    - sphinx_testing
+
+about:
+  home: "https://github.com/sphinx-doc/sphinx-testing"
+  license: "BSD"
```

Which BSD licence is it? I think it's this but please check:
```suggestion
  license: BSD-2-Clause
```